pod/perldata.pod

```diff
@@ -520,13 +522,95 @@ C<$who::0>, and a C<$who's> variable.  The last two would be the
 $0 and the $s variables in the (presumably) non-existent package
 C<who>.
 
-In fact, a simple identifier within such curlies is forced to be
-a string, and likewise within a hash subscript.  Neither need
-quoting.  Our earlier example, C<$days{'Feb'}> can be written as
-C<$days{Feb}> and the quotes will be assumed automatically.  But
-anything more complicated in the subscript will be interpreted as an
-expression.  This means for example that C<$version{2.0}++> is
-equivalent to C<$version{2}++>, not to C<$version{'2.0'}++>.
+In fact, a simple identifier within such curly braces is forced to be a
+string, and likewise within a hash subscript. Neither need quoting. Our
+earlier example, C<$days{'Feb'}> can be written as C<$days{Feb}> and the
+quotes will be assumed automatically. But anything more complicated in
+the subscript will be interpreted as an expression. This means for
+example that C<$version{2.0}++> is equivalent to C<$version{2}++>, not
+to C<$version{'2.0'}++>.
+
+There is a similar problem with interpolation with text that looks like
+an array subscript or hash subscript. Placing a simple variable like
+C<$who> immediately in front of text like C<"[1]"> or C<{foo}> would
+cause the variable to be interpolated as accesing an element of C<@who>
+or a value stored in C<%who>, and again using braces will prevent this
+from happening.
+
+    $who = "Larry Wall";
```

Comment:
This is the only piece of your feedback I am not following. The line is 72 chars long with the existing 4 spaces, and passes test. I don't want this patch to be the only part that is indented only to 1 character, there are many verbatim sections in the file, it would look weird.